RadarSDK/Radar.m

```diff
@@ -859,4 +860,54 @@ - (void)dealloc {
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
++ (BOOL)isTestKey {
+    NSString *publishableKey = [RadarSettings publishableKey];
+    if ([publishableKey hasPrefix:@"prj_test_pk"] ||
+        [publishableKey hasPrefix:@"org_test_pk"]) {
+        return YES;
+    }
+    return NO;
+}
+
++ (void)sendLog:(RadarLogLevel)level message:(NSString *_Nonnull)message {
+    [[RadarLogBuffer sharedInstance] write:level message:message];
+}
+
+/**
+ * Sends Radar log events to the server
+ */
++ (void)flushLogs {
+    // user _id has to exist
```

Comment:
hmm not sure what this comment is for

Reply:
I assume that it means that the key has to exist, and not be a test key.

Reply:
ah, when I see `user` and `_id` I'm assuming it refers to the primary key of a user entity and I didn't see a check for that. I do see the `isTestKey` check here, though

Reply:
that being said, I also wonder if the test key check should just be something we check on the server-side and not be the responsibility of the caller? this is so we can help customers debug their test builds optionally

Reply:
otherwise, the server can just drop the logs

Reply:
the test key check we wanted client side for this V1 just to be extra careful before rolling out to live implementations. was a call to minimize any risk